File: week3/phonebook.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>

typedef struct
{
    string name;
    string number;
}
person;



int main(void)
{
    person people[4];
    people[0].name = "EMMA";
    people[0].number = "654-344-8977";
    
    for (int i = 0; i < 4; i++)
    {
        if (strcmp(people[i].name, "EMMA") == 0)
        {
            printf("%s\n", people[i].number);
            return 0;
        }
    }
    printf("Not Found\n");
    return 1;
}

File: week6/phonebook.py
from sys import exit
import csv

people = {
    "GABE": 12345678,
    "LUCAS": 23456789,
    "MIKE": 34567890,
    "EMMET": 87654321
}

if "GABE" in people:
    print(f"Found {people['GABE']}")
else:
    print("Not found")

name = input("name: ")
number = input("number: ")

with open("phonebook.csv", "a") as file:

    writer = csv.writer(file)
    writer.writerow((name, number))


